charts/nri-metadata-injection/tests/volume_mounts_test.yaml

```diff
@@ -0,0 +1,21 @@
+suite: check volume mounts is properly set
+templates:
+  - templates/admission-webhooks/job-patch/job-createSecret.yaml
+  - templates/admission-webhooks/job-patch/job-createSecret.yaml
```

Comment:
NIT: duplicated file

Reply:
Thanks! Included the intended one here [18a5782](https://github.com/newrelic/k8s-metadata-injection/pull/163/commits/18a5782606f425b4bb8f0de2e0255662fad3d830)